### PR TITLE
Update "how to create a new genesis block" - Closes 1526

### LINKED ISSUE #1526: Update `How to create a new genesis block`
### Description
- Update terminology
- Add how to create genesis block assets file
- Explain how to use the genesis-block:create command
- Remove manual (by script) genesis block creation

### Additional Information
- Example: https://github.com/LiskHQ/lisk-sdk/blob/development/examples/dpos-mainchain/config/default/genesis_assets.json
- New genesis block is required when developer wants change the initial state
- If module wants to process genesis block asset, the module need to implement `initGenesisState` or `finalizeGenesisState` hook

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1526

### CODE REVIEW

File: docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc [hunk 1]
```diff
@@ -1,498 +1,98 @@
...
- "message": "process.exit"
-}
-----
-
-To fix this error, it is required to update the genesis block of the application to include the newly created account schema.
-
-== Using the CLI to generate a new genesis block
-
-The genesis block can be updated with the application CLI:
+During the init process of a sidechain client, lisk-sdk bootstraps the first genesis block required for each sidechain and the relevant assets in the `config/default` directory of each client.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc]:
```suggestion
During the init process of a sidechain client, the lisk-sdk bootstraps the first genesis block required for each sidechain and the relevant assets in the `config/default` directory of each client.
```

File: docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc [hunk 2]
```diff
@@ -1,498 +1,98 @@
...
-NOTE: For more information about the `genesis-block:create` command, check the xref:{url_references_cli_genesis}[CLI command reference].
-
-Once the above command is executed, wait until the new genesis block is generated.
-
- Creating genesis block [===========         ] 55% 44.6s
-
-When asked to overwrite the existing genesis block, reply with `yes`.
+<1> The default configuration of the sidechain client exists in the `config.json` file.
+<2> `dev-validators.json`: Contains the plain & encrypted keys, addresses, and other properties of 102 validators.
+<3> `genesis_assets.json`: Contains details about the initial state of a sidechain such as account details for `token` and `pos` modules. Also contains the list of validators, etc.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc]:
```suggestion
<3> `genesis_assets.json`: Contains details about the initial state of a sidechain such as account details for `token` and `pos` modules. It also contains the list of validators, etc.
```

File: docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc [hunk 3]
```diff
@@ -1,498 +1,98 @@
...
+<4> `genesis_block.blob`: A blob file of the genesis block.
+<5> `passphrase.json`: Contains the passphrase used in various sidechain client operations.
 
- ? A genesis_block file already exists at the given location. Do you want to overwrite it? Yes
+== Why create a new genesis block?
+For every new sidechain, a unique genesis block is required. As mentioned earlier, a default genesis block becomes available right after the client's `init` process. However, while running a public network, you may want to change the initial state of the sidechain.
 
-After the genesis block creation is complete, the following 4 files should be generated under `config/default/`:
+To do that, you can regenerate a genesis block for various use cases, such as assigning different tokens to specific addresses of your choice.  You can also recreate the genesis block based on the needs of modules, for example, setting up the initial constraints required for a staking module, etc.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc]:
```suggestion
To achieve this you can regenerate a genesis block for various use cases, such as assigning different tokens to specific addresses of your choice. Furthermore, it is also possible to recreate the genesis block based on the module's requirements, for example, setting up the initial constraints required for a staking module, etc.
```

**Review by chris529 — COMMENTED** [review]
Looks great! Just a few very minor points 👍

**Review by chris529 — APPROVED** [review]
LGTM! 👍

File: docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc [hunk 4]
```diff
@@ -1,498 +1,98 @@
...
-When asked to overwrite the existing genesis block, reply with `yes`.
+<1> The default configuration of the sidechain client exists in the `config.json` file.
+<2> `dev-validators.json`: Contains the plain & encrypted keys, addresses, and other properties of 102 validators.
+<3> `genesis_assets.json`: Contains details about the initial state of a sidechain such as account details for `token` and `pos` modules. It also contains the list of validators, etc.
+<4> `genesis_block.blob`: A blob file of the genesis block.
+<5> `passphrase.json`: Contains the passphrase used in various sidechain client operations.
 
- ? A genesis_block file already exists at the given location. Do you want to overwrite it? Yes
+== Why create a new genesis block?
+For every new sidechain, a unique genesis block is required. As mentioned earlier, a default genesis block becomes available right after the client's `init` process. However, while running a public network, you may want to change the initial state of the sidechain.
```
Comment (shuse2) [inline docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc]: for the public network, developers have to create a new genesis block. Initial genesis block are expected to be used only for development

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc [hunk 5]
```diff
@@ -1,498 +1,101 @@
...
 
- ? A genesis_block file already exists at the given location. Do you want to overwrite it? Yes
+== Why create a new genesis block?
+For every new sidechain, a unique genesis block is required. 
+As mentioned earlier, a default genesis block becomes available right after the client's `init` process. 
+However, while running a public network, the developers have to create a new genesis block to change the initial state.
+The initial genesis block is expected to be used only for the development purposes.
 
-After the genesis block creation is complete, the following 4 files should be generated under `config/default/`:
+To achieve this you can regenerate a genesis block for various use cases, such as assigning different tokens to specific addresses of your choice. Furthermore, it is also possible to recreate the genesis block based on the module's requirements, for example, setting up the initial constraints required for a staking module, etc.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc]:
```suggestion
To achieve this you can regenerate a genesis block for various use cases, such as assigning different tokens to specific addresses of your choice. 
Furthermore, it is also possible to recreate the genesis block based on the module's requirements, for example, setting up the initial constraints required for a staking module, etc.
```

**Review by shuse2 — APPROVED** [review]

File: docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc [hunk 6]
```diff
@@ -1,498 +1,102 @@
...
+<3> `genesis_assets.json`: Contains details about the initial state of a sidechain such as account details for `token` and `pos` modules. It also contains the list of validators, etc.
+<4> `genesis_block.blob`: A blob file of the genesis block.
+<5> `passphrase.json`: Contains the passphrase used in various sidechain client operations.
 
- ? A genesis_block file already exists at the given location. Do you want to overwrite it? Yes
+== Why create a new genesis block?
+For every new sidechain, a unique genesis block is required. 
+As mentioned earlier, a default genesis block becomes available right after the client's `init` process. 
+However, while running a public network, the developers have to create a new genesis block to change the initial state.
+The initial genesis block is expected to be used only for the development purposes.
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/build-blockchain/create-genesis-block.adoc]:
```suggestion
The initial genesis block is expected to be used only for development purposes.
```

**Pull request merged**